src/_playground/Routes.js

```diff
@@ -299,6 +300,12 @@ export class Routes extends Component {
         });
     };
 
+    toggleNavVis() {
```

Comment:
If you convert this to an arrow function, you won't need to bind `this` inside your `onClick`.